### ISSUE TITLE
Place Features missing from Google Drive

### ISSUE THREAD

**Issue by albertaparicio** [AnyiRao/SceneSeg#3]
I was trying to predict with the `all` configuration, using all the features and I got an error stating that the files in the `data/scene318/place_feat` directory are missing.

Once again, I checked the Google Drive directory and they are not there.

Could these features be provided in the Drive directory, or at least could I be pointed to where can I find them?

Thank you

**Comment by albertaparicio** [comment]
Also missing is the pretrained model with all modalities. In the Drive directory there are only the image-only model, a model with the audio mode and another with the place mode.

**Comment by anyirao** [comment]
Thanks for your interest! All the pre-trained models have been uploaded to Google Drive, but it might not be helpful since you don't have the features. The size of the features is kind of large. e.g. place feature takes ~5GB. It is really time-consuming to upload such a large folder with so many small files. We need to zip it and we **are uploading** it to Google Drive. Anyway, it is not fast. Another concern is that the free storage size of Google Drive is 15GB, it cannot store all the features and keyframes. We are resorting to some cloud storage services and our plan is to get everything done together with an easy-to-use toolkit in July. :)

**Comment by jdongca2003** [comment]
Hi Anyi, thank for your great work. I have a question:

In paper 4.1 (shot representation with semantic element), you mentioned "Resnet50 pretrained on places dataset (http://places.csail.mit.edu/)

In the code  ./pre/place/extract_feature.py, I observed that you used torch pretrained Resnet50 model, other than pretrained one from place dataset. 

Do you think that  image features from both models achieve similar performance?

In your Google drive (place_feat.zip), are those shot image features from torch pretrained Resnet50 model?

Thanks

Jianxiong

**Comment by anyirao** [comment]
@jdongca2003 Hi Jiangxiong,  

Thanks for your questions. We will release places365-pretrained resnet soon. These days we are kind of busy preparing camera-ready for ECCV20.

Currently, the GitHub repo uses imagenet-pretrained resnet for simplicity since PyTorch self-contains it. Otherwise, we need to find a cloud drive (not personal, belongs to the lab) to save the place365-pretrained model. Since this project is part of our big movie project, we are trying to put everything together, the releasing process is kind of complicated. We will make it public soon.

Thanks,

**Comment by jdongca2003** [comment]
Anyi,
    Thank very much for your clarification.  It is a good decision.

Based on your provided feature file (place_feature.zip, 4G) from
pretrained pytorch resnet 50, it is easy for researchers to train the
model on the provided label data and test it on new set.

You are busy now. In the future, do you plan to release audio features
(e.g. vggish)?

Thanks

Jianxiong

**Comment by zhyj3038** [comment]
Also looking forward to 3 other features left. .
For you mentioned the the total video datasets is about 10T, too large for me~~
